test/napi/src/js/functions.rs

```diff
@@ -124,11 +119,7 @@ pub fn throw_and_catch(mut cx: FunctionContext) -> JsResult<JsValue> {
 pub fn call_and_catch(mut cx: FunctionContext) -> JsResult<JsValue> {
     let f: Handle<JsFunction> = cx.argument(0)?;
     Ok(cx
-        .try_catch(|cx| {
-            let global = cx.global();
-            let args: Vec<Handle<JsValue>> = vec![];
-            f.call(cx, global, args)
-        })
+        .try_catch(|cx| f.args(()).this(cx.global()).call(cx))
```

Comment:
I think `.args` can be dropped.

```suggestion
        .try_catch(|cx| f.this(cx.global()).call(cx))
```